app/controllers/reality_check_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RealityCheckController < ApplicationController
+  def index
```

Comment:
also, interesting design choice to have a separate reality check controller, but I like it. Puts less conditional logic into the controller